shm/Player.cpp

```diff
@@ -21,3 +21,14 @@ size_t Player::countAvailableSpace() {
     }
     return ship_->getCapacity() - occupiedSpace;
 }
+
+void Player::buyCargo(Cargo* cargo, size_t amount, size_t price) {
+    ship_->load(cargo);
+    money_ = money_ - price;
+    avaliableSpace_ = avaliableSpace_ - amount;
+}
+void Player::sellCargo(Cargo* cargo, size_t amount, size_t price) {
+    ship_->unload(cargo);
+    oney_ = money_ + price;
```

Comment:
You missed one letter.
Why don't you write `money_ -= price`? It's simpler and everyone knows what does it mean.

Reply:
+1

Reply:
And regarding missing letter. Compile your code before push :)

Reply:
Hmmm... I would give my hand to chop off that the code was compiling and there was everything. I wouldn't have a hand now. My mistake -.-'